webui/react/package.json

```diff
@@ -32,11 +32,7 @@
     "unimported": "unimported"
   },
   "dependencies": {
-    "@codemirror/lang-markdown": "^6.2.5",
-    "@codemirror/lang-python": "^6.1.6",
-    "@codemirror/legacy-modes": "^6.3.3",
     "@glideapps/glide-data-grid": "^6.0.3",
-    "@uiw/react-codemirror": "^4.22.00",
```

Comment:
we moved unnecessary dependencies. code mirror deps are installed through hew library